jwst/datamodels/__init__.py

```diff
@@ -130,11 +131,12 @@
     'ReferenceFileModel', 'ReferenceCubeModel', 'ReferenceImageModel', 'ReferenceQuadModel',
     'RegionsModel', 'ResetModel',
     'ResolutionModel', 'MiriResolutionModel',
-    'RSCDModel', 'SaturationModel', 'SpecModel',
+    'RSCDModel', 'SaturationModel', 'SlitModel', 'SpecModel',
```

Comment:
👍  for maintaining the alphabetical order of model definitions! This is the really important stuff.